### ISSUE TITLE
`genie_n_pls_m_gs` is generating linear injections

### ISSUE THREAD

**Issue by zeeshan5885** [gwkokab/gwkokab#275]
### Check for existing issues

- [X] Completed

### Description

It seems like the issue https://github.com/gwkokab/gwkokab/issues/263 got fixed and I am not getting a flat local acceptance rate. However, I am using the following script to generate the data for a single power-law.

```
genie_n_pls_m_gs \
    --vt_path "../../../vts/vt_1_200_ecc_matters.eqx" \
    --analysis_time 50 \
    --error_size 4000 \
    --num_realizations 1 \
    --rate 1000 \
    --model-json model.json \
    --err-json err.json

```

but it is giving me only linear injections. 
![m1m2_inj_0](https://github.com/user-attachments/assets/b3a95d9d-6f74-48f7-982d-90e5e31fc98f)

**Comment by zeeshan5885** [comment]
Another similar plot on different values of alpha=-2  and beta=2

![m1m2_inj_0-2](https://github.com/user-attachments/assets/63aabbd6-98c4-4a9d-aff8-4ed9ae9dcffd)

**Comment by zeeshan5885** [comment]
- There is something strange going on.
- The following plot is for the model `genie_ecc_matters` and it is also giving me linear injections.

![m1m2_inj_0](https://github.com/user-attachments/assets/ff8efc4f-1ec4-4b07-a7fd-df7f32562474)

**Comment by zeeshan5885** [comment]
single and double Gaussian are giving me reasonable injections.
![m1m2_inj_0-3](https://github.com/user-attachments/assets/ed248194-2954-4f0f-8ae5-fda129f777e8)


![m1m2_inj_0-2](https://github.com/user-attachments/assets/d743b4f2-d50d-43eb-9807-afcde28d4981)
0-4b89-8ed4-c12e82da0b41)

**Comment by Qazalbash** [comment]
The problem is we have used the same key to generate different random variates.

https://github.com/gwkokab/gwkokab/blob/d7774196e0c262b82c70f5b03b34e9f9fb1faf5b/src/gwkokab/models/_models.py#L810-L818

**Comment by Qazalbash** [comment]
Fixed by #276.
